controllers/subscriptions.go

```diff
@@ -22,6 +22,13 @@ type getter func(string) []string
 
 var cache = ccache.New(ccache.Configure().MaxSize(500).ItemsToPrune(50))
 
+var bundleInfo []types.Bundle
+
+func init() {
+	// Reads the yaml file at process startup
+	bundleInfo = GetBundleInfo(config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml))
```

Comment:
Is is valid to start the service if you end up with no bundles?

Reply:
I was playing around with the code a bit. It's quite simple to have entitlements return something like `{"":{"is_entitled":true}}`, when `bundles.yml` is unavailable. However, I am not sure if I should pursue that. Definitely need some directions here... :disappointed:

Reply:
Sorry, I probably approached it the wrong way 🤦‍♂ 
I think that's a very good question, and maybe something we can address in a future PR